handbook/Best practices/TypeScript/Project-References.md

```diff
@@ -0,0 +1,166 @@
+# Project References
+
+## Table of contents
+
+1. [What are Project References?](#what-are-project-references)
+1. [Migrating a codebase](#migrating-a-codebase)
+1. [Tips](#tips)
+1. [Caveats](#caveats)
+
+## What are Project References?
+
+[Project References](https://www.typescriptlang.org/docs/handbook/project-references.html) provides built-in scalability for TypeScript. 
```

Comment:
It might be nice to provide a one-line description here too as a tl;dr, something like the one from the link provided:

> Project references are a new feature in TypeScript 3.0 that allow you to structure your TypeScript programs into smaller pieces.

> By doing this, you can greatly improve build times, enforce logical separation between components, and organize your code in new and better ways.